src/Project.ts

```diff
@@ -207,21 +232,25 @@ export default class Project {
       triggers.map(({ trigger, target }) => {
         return trigger.start(target);
       })
-    );
+    ).then();
```

Comment:
This is really weird. I get it, but can we do an async function instead?

Also unnecessary `() => { return x }` pattern above.

Reply:
Changed to an async function